### PR TITLE
Added frame type movement_control

### PR DESCRIPTION
**Pull request by MarkGasse**

### CODE REVIEW

File: code/headers/frame_types.hpp [hunk 1]
```diff
@@ -240,10 +241,32 @@ namespace r2d2 {
...
+     * 
+     * Moving Platform wiki:
+     * https://github.com/R2D2-2019/R2D2-2019/wiki/Moving-Platform
+     */
+    struct frame_movement_control_s {
+        // A value between -100% & 100% 
+        int8_t speed;
+
+        // A value between -90 & 90 (degrees)
+        int8_t rotate;
```
Comment (LRstudentHU) [inline code/headers/frame_types.hpp]: `rotation` would probably be better.

File: code/headers/frame_types.hpp [hunk 2]
```diff
@@ -60,6 +60,7 @@ namespace r2d2 {
         DISPLAY_FILLED_RECTANGLE,
         BATTERY_LEVEL,
         UI_COMMAND,
+        MOVEMENT_CONTROL,
```
Comment (LRstudentHU) [inline code/headers/frame_types.hpp:63]: `MOVEMENT_CONTROL` or `MANUAL_CONTROL`?

**Review by LRstudentHU — CHANGES_REQUESTED** [review]

File: code/headers/frame_types.hpp (hunk 2 above)
Comment (MarkGasse) [reply code/headers/frame_types.hpp:63]: At first we called it MANUAL_CONTROL, but this frame could also be used to move the moving platform without manual control. So we decided MOVEMENT_CONTROL would be a better name.

File: code/headers/frame_types.hpp (hunk 1 above)
Comment (MarkGasse) [reply code/headers/frame_types.hpp]: I am going to rename it. Thanks

**Review by LRstudentHU — APPROVED** [review]

**Pull request merged**